unlock-protocol.com/blog/digital-cash.md

```diff
@@ -0,0 +1,48 @@
+</p>
+
+# Digital Cash
+
+Blockchain transactions behave more like cash. We have a mutual understanding of the value of our physical money and you own and control the cash that’s in your real world wallet. You hand over the cash without anyone in middle and the transaction is done. This is very similar to the blockchain with a few additional advantages: your blockchain wallet can live in several places and you can transact without having to be in the same physical space. On  top of that the Ethereum blockchain (and other blockchain projects) let you create smart contracts that allow you to codify and automate parts of the transaction.
+
+I liked this analogie by Steven Chien: he calls the internet the “Information Superhighway” – which allowed for the large scale (and cost efficient) distribution of information and the blockchain the “Value Superhighway” – that allow for a large scale and cost efficient but also more equitable distribution of value. It’s a good read, check it out on Hackernoon.
+
+<p style="text-align:center">
+	<img src="/static/images/blog/digital-cash/keychain.jpg" width="400px" alt="Towards an access ecosystem.">
+</p>
+
+# Access
+
+So that’s cool, we have a wallet, we control what’s in it, how we transact and there are no intermediaries. Its also relatively fast all things considered: blockchain transactions sometimes take a while to confirm but in terms of overall settlement its faster than a bank transfer or a credit card transaction (they can take anywhere from 3 to 30 days to complete).
+
+Because there's a ledger of every transaction, apps can easily determine who’s paid for access and unlock it for you. In that sense your wallet is also a keychain.
+
+
+<p style="text-align:center">
+	<img src="/static/images/blog/digital-cash/creditcard.jpg" width="400px" alt="One wallet, access to many different things.">
+</p>
+
+# Beyond Cash
+Taking that idea a step further you could start swapping, trading or even bundling keys where one key gives you access to multiple locks (I wrote about this in our Multi Use Locks post if you’re curious about that). The most interesting part is yet to come: ledger plus smart contracts allows users to receive tokens for specific actions – so if you refer a creators content or you’ve paid for access 10 times you could get a kickback.
+
+This feature set is really powerful: imagine if you could get into your WeWork office with your credit card (because you’ve paid for your desk with it) but it also gives you access to the new york times online, gets you bonus points at your local coffee shop and at the end of the day opens up and pays for your car2go or zipcar.
+
+You have thoughts or comments? Reach out I’d love to chat. [@supermombartz](https://twitter.com/supermombartz) or [sascha@unlock-protocol.com](mailto:sascha@unlock-protocol.com) or become a member of the blog and add a comment.
```

Comment:
Not sure it is a good idea to point to "personal" accounts or email adresses, let's use `hello@...` so that we can easily redirect traffic at any point ;)

Reply:
wanted to keep it personal but sure